src/Stripe.net/Entities/Sources/SourceBecsDebit.cs

```diff
@@ -0,0 +1,19 @@
+namespace Stripe
```

Comment:
This should be named `SourceAuBecsDebit` since that's the name of the whole type.